src/main.cpp

```diff
@@ -117,7 +117,7 @@ void configureApp(bool gui)
 
 // TODO find a way so we don't have to do this
 /// Recreate the application as a QApplication
-void reinitializeAsQApplication(int argc, char* argv[])
+void reinitializeAsQApplication(int &argc, char* argv[])
```

Comment:
@marceldev89 Nice catch. Though it looks to me that `argv` could fail in the same way. Can you replace it with `char *(&argv)[]`?

Reply:
@veracioux it shouldn't. The problem was that the `QApplication` constructor signature which is `QApplication(int &argc, char **argv)`. So it expects a reference to `argc` instead of the value. `argv` should be fine as `char* argv[]` is basically the same as `char **argv` I think :)

Reply:
@marceldev89 Yup, very subtle and annoying. Please do a [clang-format](https://github.com/flameshot-org/flameshot/blob/master/docs/CONTRIBUTING.md#code) on the code, so our pipeline can pass before merge.

Reply:
Done :)

Reply:
Thank you! Merging.